src/Std.dfy

```diff
@@ -0,0 +1,52 @@
+module Std {
+
+  newtype uint8 =  x | 0 <= x < 0x100
```

Comment:
Will this `uint8` type be used instead of an unsigned `byte`?

Reply:
Yes, that's what I would suggest, because if you just say `byte`, it's not clear if it's signed or unsigned.